.github/workflows/podvm_smoketest.yaml

```diff
@@ -57,11 +59,11 @@ jobs:
 
       - name: Build image
         run: make image
-     
+
       - name: Install kata-agent-ctl
         run: |
           oras pull "${KATA_REG}/agent-ctl:${KATA_REF}-x86_64"
-          tar xf kata-static-agent-ctl.tar.xz 
+          tar xf kata-static-agent-ctl.tar.xz
```

Comment:
This is an unrelated change, could you please remove it (in case we need to do a revert we want to keep this change separate)